cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ShowProceduresTest.scala

```diff
@@ -28,60 +28,158 @@ class ShowProceduresTest extends DocumentingTest {
   override def doc: Document = new DocBuilder {
     doc("SHOW PROCEDURES", "query-listing-procedures")
     p("Listing the available procedures can be done with `SHOW PROCEDURES`.")
-    p("include::listing-procedures-syntax.asciidoc[]")
     p("This command will produce a table with the following columns:")
-    p("include::listing-procedures-columns.asciidoc[]")
-    section("Examples") {
-      section("Listing all procedures") {
-        p(
-          """To list all available procedures with the default output columns, the `SHOW PROCEDURES` command can be used.
-            #If all columns are required, use `SHOW PROCEDURES YIELD *`.""".stripMargin('#'))
-        query("SHOW PROCEDURES", ResultAssertions(p => {
-          p.columns should contain theSameElementsAs Array("name", "description", "mode", "worksOnSystem")
-        })) {
-          limitedResultTable(List("name", "description", "mode", "worksOnSystem"), 15)
-        }
+    p("""
+[NOTE]
+====
+The command `SHOW PROCEDURES` only output the default output; for a full output use the optional `YIELD` command.
```

Comment:
```suggestion
The command `SHOW PROCEDURES` only outputs the default output; for a full output use the optional `YIELD` command.
```
Also, as above, I'd put the admonition right after _Listing the available procedures can be done with SHOW PROCEDURES._